src/Laravel/PhpDatatypesServiceProvider.php

```
@@ -0,0 +1,88 @@
+            'int32' => 'The :attribute must be a valid 32-bit signed integer.',
+            'int64' => 'The :attribute must be a valid 64-bit signed integer.',
+            'uint8' => 'The :attribute must be a valid 8-bit unsigned integer (0 to 255).',
+            'uint16' => 'The :attribute must be a valid 16-bit unsigned integer (0 to 65,535).',
+            'uint32' => 'The :attribute must be a valid 32-bit unsigned integer.',
+            'uint64' => 'The :attribute must be a valid 64-bit unsigned integer.',
+            'float32' => 'The :attribute must be a valid 32-bit floating point number.',
+            'float64' => 'The :attribute must be a valid 64-bit floating point number.',
+        ];
+
+        foreach ($messages as $rule => $message) {
+            Validator::replacer($rule, function ($message, $attribute, $rule, $parameters) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$parameters'. (undefined)

(UnusedFormalParameter)